health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationRedesign/DrawerFieldComposer.js

```
@@ -126,6 +127,37 @@ const RenderField = ({ state, panelItem, drawerState, setDrawerState, updateLoca
               />
             )
           ) : null}
+          {shouldShow && shouldShow?.showErrorMessage ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)